ste/downloader-blob.go

```diff
@@ -66,9 +66,28 @@ func newBlobDownloader(jptm IJobPartTransferMgr) (downloader, error) {
 		return nil, err
 	}
 
+	blobClient := s.NewContainerClient(jptm.Info().SrcContainer).NewBlobClient(jptm.Info().SrcFilePath)
```

Comment:
it might be better to pass this as a variable in jptm.Info() (like SrcContainer and SrcFilePath) 

See mgr-JobPartTransferMgr line 266 for that method

Reply:
I think this is probably a bug in more places than just here too, can you look into where else we may have missed snapshot/version id setting?

Reply:
resolved in new pr https://github.com/Azure/azure-storage-azcopy/pull/2527